src/entity/active_enum.rs

```diff
@@ -275,6 +275,64 @@ mod tests {
         assert_eq!(format!("{}", DeriveCategory::Small), "'S'");
     }
 
+    #[test]
+    fn display_mode() {
+        // Completely default -> All print values
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer")]
+        enum DefaultDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", DefaultDisplay::One), "1");
+        assert_eq!(format!("{}", DefaultDisplay::Two), "2");
+        assert_eq!(format!("{}", DefaultDisplay::Three), "3");
+
+        // All `label` -> All print label
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer", display = "label")]
+        enum LabelDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", LabelDisplay::One), "'One'");
```

Comment:
Umm... I don't have a clear sense, but seems like the `'` is unnecessary in this context? Like the expected should be:

```rust
assert_eq!(format!("{}", LabelDisplay::One), "One");
```

Reply:
that relied on the implementation of `Value::to_string`, so...

Reply:
as discussed, we will keep the behaviour